README.md

```diff
@@ -6,11 +6,9 @@ Apollo
 [![Maven Central](https://img.shields.io/maven-central/v/com.spotify/apollo-parent.svg)](https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.spotify%22%20apollo*)
 [![License](https://img.shields.io/github/license/spotify/apollo.svg)](LICENSE.txt)
 
-## Status: Bug-fix only
-
-Apollo is heavily used within Spotify, however, most of its development is currently done internally leveraging Apollo's module system. We want to signal this current state to the community by putting the project in maintenance mode.   
-This project will no longer have new features or accept PRs for new features. We will continue to accept bug fixes and update dependencies. 
+## Status: Archived
 
+> ⚠️ Apollo is heavily used within Spotify, however, most of its development is currently done internally leveraging Apollo's module system. We want to signal this current state to the community by putting the project in archive mode. This means this project will no longer have new features, bug fixes or accept PRs for new features. Apollo development will continue internally in Spotify.
```

Comment:
```suggestion
> ⚠️ Apollo is heavily used within Spotify, however, most of its development has recently been done internally leveraging Apollo's module system. Because of this, and the fact that Apollo has very little (if any) use outside Spotify we want to signal this current state to the community by putting the project in archive mode. This means this project will no longer have new features, bug fixes or accept PRs for new features. Apollo development will continue internally in Spotify.
```